docs/GettingStartedDocs/Contributors/WindowsInstallInfo.md

```diff
@@ -42,3 +42,13 @@ validate remote reports from enclaves.
 ```cmd
 cpack -D CPACK_NUGET_COMPONENT_INSTALL=ON -DCPACK_COMPONENTS_ALL=OEHOSTVERIFY
 ```
+
+## Install the NuGet package
+
+Before the `.nupkg` packages can be installed, they must be moved into a separate directory. Assuming this directory is `C:\oe_nuget\`, then the installation procedure is as followed:
+
+```cmd
+Nuget.exe install "open-enclave" -OutputDirectory C:\oe -ExcludeVersion -Source C:\oe_nuget\
+```
+
+The package is installed at `C:\oe\open-enclave\openenclave`. The installation can then be tested by building and running samples with `CMAKE_PREFIX_PATH` set to `C:\oe\open-enclave\openenclave\lib\openenclave\cmake\`
```

Comment:
Was this with the extra `cpack` step?

Reply:
Oops somehow the `&& cpack` were removed. I added them back